src/sentry/notifications/notifications/rules.py

```diff
@@ -166,6 +166,20 @@ def send(self) -> None:
                         recipient_email=participant.email,
                         recipient_username=participant.username,
                     )
+                if (
+                    features.has("organizations:active-release-monitor-alpha", self.organization)
+                    and last_release
+                ):
+                    ReleaseActivity.objects.create(
+                        type=ReleaseActivity.Type.issue,
+                        data={
+                            "provider": EXTERNAL_PROVIDERS[provider],
+                            "issue_id": self.group.id,
+                            "issue_error_count": self.group.times_seen,
+                            "issue_affected_users": self.group.count_users_seen(),
```

Comment:
not sure the error_count and issue_affected_users is worth saving

Reply:
If we return the serialized group instead of `issue_id`, `issue_error_count` and `issue_affected_users`, would that still work with the current design?

Reply:
i have it as `data.group`, but whatever works

Reply:
Ok, that works for me. I added a thing.